src/__tests__/role.js

```diff
@@ -26,7 +26,7 @@ test('by default logs accessible roles when it fails', () => {
 })
 
 test('when hidden: true logs available roles when it fails', () => {
-  const {getByRole} = render(`<div hidden><h1>Hi</h1></div>`)
+  const {getByRole} = render(`<h1 hidden>Hi</h1>`)
```

Comment:
Let's keep this test. I don't think it's relevant to the change. If there is something important here, lets's add a new test instead so that we can better track what changed

Reply:
The idea was to test the case where an element had a name available to be filtered but is hidden, before that change the name was always empty. May we can add a test specifically for that case? What do you think?

Reply:
Always add test instead of re-purposing existing tests. Existing tests should only change their assertions if they relied on buggy behavior.

Reply:
Thank you! I'm learning about all that tests concepts and patterns.

Reply:
@eps1lon can you check this again? :3